packages/app/ui/containers/Tides.js

```diff
@@ -2,36 +2,196 @@ import React from 'react';
-    tides(lat: $lat, lon: $lon, timezone: $timezone) {
-      dt
-      date
-      height
-      type
+  query($spotId: ID!) {
+    tides(spotId: $spotId){
+      spot {
+        id
+      }
+      currentTime {
+        timezone
+        hours
+        minutes
+        prettyTimeLabel
+      }
+      tides {
+        currentTide {
+          type
+          dt
+        }
+        tomorrow {
+          dt
+          date
+          prettyTimeLabel
+          prettyDateTimeLabel
+          day
+          month
+          year
+          height
+          type
+        }
+        today {
+          dt
+          date
+          prettyTimeLabel
+          prettyDateTimeLabel
+          day
+          month
+          year
+          height
+          type
+        }
+        allTides {
+          dt
+          date
+          prettyTimeLabel
+          prettyDateTimeLabel
+          day
+          month
+          year
+          height
+          type
+        }
+      }
     }
   }
 `;
 
+function renderPane(data, currentTide=null, currentTime=null) {
+  return (
+    <Tab.Pane attached='bottom'>
+      <style jsx>{`
+        .highTide, .lowTide {
+          text-align: center;
+        }
+        .highTide {
+         padding-top: 50px;
+        }
+
+        .lowTide {
+          padding-top: 100px
+        }
+      `}</style>
+      <section>
+        <p><b>{data[0].prettyDateTimeLabel} {currentTime ? currentTime.prettyTimeLabel : null}</b></p>
+        <p>
+        {
+          currentTide ? 
+            currentTide.type === 'Rising' ? 
+              `${currentTide.type} ↑` :
+              `${currentTide.type} ↓`
+            : null
+        }
+        </p>
+        <Grid columns={data.length}>
+          <Grid.Row>
+            {data.map(({ dt, prettyTimeLabel, height, type }) => (
+              <Grid.Column key={dt}>
+                <div className={type === 'High' ? 'highTide' : 'lowTide'}>
+                  {prettyTimeLabel}
+                  <hr/>
+                  {height}m
+                </div>
+              </Grid.Column>
+            ))}
+          </Grid.Row>
+        </Grid>
+      </section>
+    </Tab.Pane>
+  )
+}
+
+function renderTableTides(tides, key) {
+  return (
+    <React.Fragment key={key}>
+    {
+      _.map(tides, (t, k) => (
+        <tr key={k}>
+          <td>{t.prettyTimeLabel}</td>
+          <td>{t.height}</td>
+          <td>{t.type}</td>
+        </tr>
+      ))
+    }
+    </React.Fragment>
+  )
+}
+
+function renderAllTidesPane(data) {
+  const allDates = _.uniqBy(data, 'prettyDateTimeLabel');
+  return (
+    <Tab.Pane attached='bottom'>
+      <style jsx>{`
+        dateTitle: {
+          margin-top: 40px!important;
+        }
+      `}</style>
+      <section>
+        {
+          _.map(allDates, (d, k) => {
+            return (
+              <div key={k}>
+                <h2 style={{marginTop: 40}} className="dateTitle">{d.prettyDateTimeLabel}</h2>
```

Comment:
please don't style things inline, let's keep this clean and reuse Semantic's alignment and flow as much as possible